docs/registry.md

```diff
@@ -113,14 +113,14 @@ to add your _json_key authentication for gcr.io domain image pull
 requests:
 
 ```
-[plugins."io.containerd.grpc.v1.cri".registry]
```

Comment:
We recommend the version 2 syntax.

Please add `version = 2` header explicitly and revert other changes.

Reply:
good point line 15 should have the "version = 2" text

Reply:
Thanks!
I have some unknown part yet.
So I'll try understand and submit the changes about that part tomorrow.